tools/infrasys/src/main.rs

```diff
@@ -62,7 +63,17 @@ fn run() -> Result<()> {
     // Parse and store the args passed to the program
     let args = Args::from_args();
 
-    SimpleLogger::init(args.log_level, LogConfig::default()).context(error::LoggerSnafu)?;
+    match args.log_level {
+        // Set log level for AWS SDK to error to reduce verbosity.
+        LevelFilter::Info => Builder::new()
+            .filter_level(args.log_level)
+            .filter(Some("aws_config"), LevelFilter::Error)
+            .filter(Some("aws_smithy"), LevelFilter::Error)
```

Comment:
In `pubsys` we set this to `LevelFilter::Warn` - should we do that here for consistency?

Reply:
Fixed